### ISSUE TITLE
OTR Sidebar Extended View

### ISSUE THREAD

**Issue by nemaniarjun** [RocketChat/Rocket.Chat#10316]
When in an OTR conversation, the sidebar shows the encrypted JSON string.
It should instead show something like "Encrypted Message".

**Comment by nemaniarjun** [comment]
![image](https://user-images.githubusercontent.com/15715894/38268065-d35eba66-379a-11e8-9586-97d93739d258.png)

**Comment by TwizzyDizzy** [comment]
While your issue is valid, the format of your issue is not. Please adhere to the [issue template](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/ISSUE_TEMPLATE.md) and add these information in your initial post.

Cheers
Thomas

**Comment by nemaniarjun** [comment]
Sorry. Updated :)

**Comment by TwizzyDizzy** [comment]
Thank you :) As it's probably saved encrypted there (where usually in mongodb there would be the cleartext phrase in non-OTR conversations) I think it's not thaaaat bad... on the other hand there is an information left in the DB simply that *there was* an actual conversation which doesn't align with the goals of OTR, I think (deniability).

Just my thoughts...

Cheers
Thomas

**Comment by nemaniarjun** [comment]
@TwizzyDizzy I dont think it is a security issue. Instead I think it is just a small bug, non urgent but easy to fix. I will create a PR soon. :)

**Comment by KevLehman** [comment]
OTR feature was removed on latest version.
